packages/headless/src/features/folding/folding-actions.ts

```diff
@@ -23,3 +44,41 @@ export const registerFolding = createAction(
   (payload: RegisterFoldingPayload) =>
     validatePayload(payload, foldingOptionsSchemaDefinition)
 );
+
+export const loadAll = createAsyncThunk<
+  LoadAllFulfilledReturn,
+  CollectionId,
+  AsyncThunkSearchOptions<
+    ConfigurationSection & FoldingSection,
+    LoadAllRejectedReturn
+  >
+>(
+  'folding/loadAll',
+  async (
+    collectionId: CollectionId,
+    {getState, rejectWithValue, extra: {searchAPIClient}}
+  ) => {
+    const {
+      folding: {fields},
+      configuration: {
+        accessToken,
+        organizationId,
+        search: {apiBaseUrl},
+      },
+    } = getState();
+
+    const response = await searchAPIClient.search({
+      accessToken,
+      organizationId,
+      url: apiBaseUrl,
+      aq: `@${fields.collection} = ${collectionId}`,
+      numberOfResults: 100,
```

Comment:
The current implementation fetches up to 100 results and won't allow more results to be fetched.

Alternatively, we could make this default of 100 customizable or we could change "loadAll" into "fetchMore" and fetch less results multiple times.

Reply:
Let's keep it fixed to 100 for now,